regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -90,8 +90,8 @@ This contains all of the styles specific to the TOC
 
   a:link,
   a:visited {
-    color: @black;
-    .border-bottom-light (@width: 1px);
+    color: @navigation_gray;
+    .border-bottom-light (@width: 0px);
```

Comment:
This border is still expected.

ATF current
<img width="228" alt="screen shot 2016-06-22 at 7 16 10 pm" src="https://cloud.githubusercontent.com/assets/326918/16286966/de9a941a-38ad-11e6-828d-ff3eca14cfdf.png">


Patch
<img width="231" alt="screen shot 2016-06-22 at 7 16 18 pm" src="https://cloud.githubusercontent.com/assets/326918/16286964/dc2213fc-38ad-11e6-8cf4-db3e4e5a0206.png">